html-web-page/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+
+      .hide {
+        /*
+        copy pasted from w3 🤷‍♂️
+        https://www.w3.org/WAI/tutorials/forms/labels/
+        basically turns it into a 1px by 1px box
+        */
+        border: 0;
+        clip: rect(0 0 0 0);
+        height: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute;
+        width: 1px;
+        visibility: hidden;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+    <p>
+      Here at JOTMC Inc., we are constantly striving to supply our customers
+      with the highest-quality and most unique jellies on a monthly basis. For
+      the jelly lover in your life (or yourself!), this is the gift that keeps
+      on giving all year round. We offer three tiers of monthly jelly joy, and
+      each one offers their own perks. Here is a breakdown of each, as well as
+      your monthly price:
+    </p>
+    <table>
+      <thead>
+        <tr>
+          <th></th>
+          <th>Basic Plan</th>
+          <th>Deluxe Plan</th>
+          <th>Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&#10004;</td>
+          <td>&#10004;</td>
+          <td>&#10004;</td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&#10004;</td>
+          <td>&#10004;</td>
+        </tr>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td>&#10004;</td>
+          <td>&#10004;</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&#10004;</td>
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+    <form action="">
+      <label for="username" class="hide">Your Name</label>
+      <input
+        type="text"
+        id="username"
+        placeholder="Your Name"
+        name="username" />
+      <label for="plan" class="hide">Your Plan</label>
```

Comment:
I like how you did a research in order to hide the label! This totally works but when we don't need `label` element to actually "label" the inputs, we simply just don't need to add it at all.

Example (no label needed so `id` is no longer needed as well):
`<input type="text" placeholder="Your Name" name="username" />`

Having said that, let's do some cleanup and delete labels with `hide` class and remove `hide` css styling! Great job btw!

Reply:
Ok, that makes things easier then 😅
I'll go ahead and fix that right now